public/index.proto.html

```diff
@@ -6,9 +6,25 @@
   <link rel="stylesheet" href="docsupport/style.css">
   <link rel="stylesheet" href="docsupport/prism.css">
   <link rel="stylesheet" href="chosen.css">
+
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self' https://ajax.googleapis.com; style-src 'self' 'sha256-MzlZlLruPmzLm/yy8z6yCpmyo5QSe/SURdcVrrJx880='; img-src 'self' data:">
```

Comment:
Even though calculating the hash is easy, because the correct one is logged to the console when using an incorrect hash, the forgetting part is true. 

But then again, a nonce isn't safe if not regenerated on every request. Not that there's really a security risk at this page, I still think we shouldn't advertise the less secure option.

Maybe we can move the CSP additions to the build process of the documentation, and leave out the header in the HTML here.

Or validate the HTML in the build process, if possible.